src/main/java/uk/gov/pay/ledger/transaction/resource/TransactionResource.java

```diff
@@ -100,6 +104,45 @@ public TransactionSearchResponse search(@Valid
         return transactionService.searchTransactionsForCsv(searchParams);
     }
 
+    @Path("/stream")
+    @GET
+    @Produces("text/csv")
+    @Timed
+    public Response streamCsv(@Valid @BeanParam TransactionSearchParams searchParams,
+                              @QueryParam("override_account_id_restriction") Boolean overrideAccountRestriction,
+                              @QueryParam("account_id") String gatewayAccountId,
+                              @Context UriInfo uriInfo) {
+        StreamingOutput stream = outputStream -> {
+            TransactionSearchParams csvSearchParams = Optional.ofNullable(searchParams).orElse(new TransactionSearchParams());
```

Comment:
Might be clean if we can move this logic into a service like `CsvStreamService` or similar

Reply:
Considering relationship between resource level and streaming, for now moving anything not to do with streaming into csv service